selftests/unit/test_multiplexer.py

```diff
@@ -39,7 +39,7 @@ def test_basic(self):
         self.assertNotEqual(self.params1, self.params2)
         repr(self.params1)
         str(self.params1)
-        str(multiplexer.AvocadoParams([], 'Unittest', [], {}))
+        str(variants.AvocadoParams([], 'Unittest', [], {}))
```

Comment:
OK, although I still find a few references to the term "multiplexer".  Hopefully this would be helpful for you to check if they make sense or not:

```
$ git grep multiplexer 
README.rst:- A multiplexer that allows tests to be run with different sets of variables;
avocado/core/runner.py:        :param mux: the multiplexer.
avocado/plugins/diff.py:            return ["Unsupported version of multiplexer: %s" % details]
docs/source/Configuration.rst:* multiplexer
docs/source/Mux.rst:The ``Mux`` object is defined in ``avocado/core/multiplexer.py``, is always
docs/source/Mux.rst:by path. But the most powerful multiplexer feature is that it can generate
docs/source/Mux.rst:time.In order to generate all possible variants multiplexer creates cartesian
docs/source/ReferenceGuide.rst:The multiplexer component creates different sets of variables
docs/source/ReferenceGuide.rst:multiplexer to identify each variant. It should be unique per
docs/source/ReferenceGuide.rst:variant inside a set. In other words, the multiplexer generates a
docs/source/ReferenceGuide.rst:A simpler implementation of the multiplexer uses serial integers
docs/source/ReferenceGuide.rst:.. note:: The multiplexer supports serialized variant IDs only
docs/source/WritingTests.rst:   2016-11-02 11:13:01,394 multiplexer      L0166 DEBUG| PARAMS (key=timeout, path=*, default=None) => 3
docs/source/WritingTests.rst:   2016-11-02 11:13:01,396 multiplexer      L0166 DEBUG| PARAMS (key=sleep_length, path=*, default=1) => 5
docs/source/WritingTests.rst:If you pass that multiplex file to the runner multiplexer, this will register
```